src/pages/People.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import { Loader } from '../components/Loader';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { PersonLink } from '../components/PersonLink';
+
+export const People: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [isLoading, setIsLoading] = useState(true);
+  const [isError, setIsError] = useState(false);
+
+  useEffect(() => {
+    const gettingPeople = async () => {
+      try {
+        const peopleFromServer = await getPeople();
+
+        const peopleWithParents = peopleFromServer.map(person => ({
+          ...person,
+          father: peopleFromServer.find(p => p.name === person.fatherName),
+          mother: peopleFromServer.find(p => p.name === person.motherName),
```

Comment:
The logic for finding a person's father and mother by name may lead to incorrect associations if there are multiple people with the same name. Consider using a unique identifier for each person to ensure accurate parent-child relationships.